.buildkite/scripts/steps/esql_grammar_sync.sh

```diff
@@ -118,11 +118,10 @@ main () {
 
   report_main_step "Changes committed. Creating pull request."
 
-  git remote add kibanamachine https://github.com/kibanamachine/kibana.git
-  git push kibanamachine "$BRANCH_NAME"
+  git push origin "$BRANCH_NAME"
 
   # Create a PR
-  gh pr create --draft --title "$PR_TITLE" --body "$PR_BODY" --base main --head "kibanamachine:${BRANCH_NAME}" --label 'release_note:skip' --label 'Team:ESQL' 
+  gh pr create --draft --title "$PR_TITLE" --body "$PR_BODY" --base main --head "${BRANCH_NAME}" --label 'release_note:skip' --label 'Team:ESQL' 
```

Comment:
Was dropping kibanamachine intentional?

Reply:
Yes, we no longer go through a fork `kibanamachine/kibana` but we push the branch to `elastic/kibana` (the origin) and create the pr from there. The new ephemeral token issuing doesn't work for forks (see: https://github.com/elastic/ci/pull/2805)